hotels.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Гостиницы. HTML Academy: Седона. </title>
+  </head>
+  <body>
+
+    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
+    <header>
+      <nav>
+        <a href="index.html">
+          <img src="" width="" height="" alt="Логотип SITY OF SEDONA">
+        </a>
+        <ul>
+          <li><a href="blank.html">Информация</a></li>
+          <li><a href="blank.html">Фото и видео</a></li>
+          <li><a href="blank.html">Карта штата</a></li>
+          <li><a href="hotels.html">Гостиницы</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+
+        <h1>SITY OF SEDONA Гостиницы</h1><!--скрыть ч/з visually hidden -->
+        <section>
+          <h2>Фильтр гостиниц</h2><!--скрыть ч/з visually hidden -->
+          <!--Здесь будет форма фильтра-->
+        </section>
+
+        <section>
+          <h2>Cписок гостиниц</h2><!--скрыть ч/з visually hidden -->
+          <ul>
+            <li>
+              <a href="blank.html">
+                <p>
+                  <img src="" width="" height="" alt="Фото Amara Resort & Spa" >
+                </p>
+                <h3>Amara Resort & Spa</h3><!--text-transform: uppercase;(весь текст сайта)-->
+                <!--Здесь будет описание гостиницы в виде таблицы (со ссылкой и кнопкой)-->
+              </a>
+            </li>
+            <li>
+              <a href="blank.html">
+                <p>
+                  <img src="" width="" height="" alt="Фото Desert Quail Inn" >
+                </p>
+                <h3>Desert Quail Inn</h3><!--text-transform: uppercase;(весь текст сайта)-->
+                <!--Здесь будет описание гостиницы в виде таблицы (со ссылкой и кнопкой)-->
+              </a>
+            </li>
+            <li>
+              <a href="blank.html">
+                <p>
+                  <img src="" width="" height="" alt="Фото Villas at Poco Diablo" >
+                </p>
+                <h3>Villas at Poco Diablo</h3><!--text-transform: uppercase;(весь текст сайта)-->
+                <!--Здесь будет описание гостиницы в виде таблицы (со ссылкой и кнопкой)-->
+              </a>
+            </li>
+          </ul>
+        </section>
+
+    </main>
+
+
+
+
+
+    <footer>
+      <section>
+        <h2>Телефон</h2><!--скрыть ч/з visually hidden -->
+        <p><strong><a href="#">#</a>visitsedona<br>
```

Comment:
Если делать ссылкой, то тогда уже весь текст:
```suggestion
        <p><strong><a href="#">#visitsedona</a><br>
```